tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -90,6 +90,7 @@ def get_message_subject(sqs_message):
 def setup_defaults(config):
     config.setdefault('region', 'us-east-1')
     config.setdefault('ses_region', config.get('region'))
+    config.setdefault('runtime', 'python2.7' if str is bytes else 'python3.6')
```

Comment:
can we make this explicit on the runtime selection

Reply:
Soooo, mixing runtimes between local and remote leads to inconsistencies ([e.g.](https://github.com/whit537/cloud-custodian/pull/37#issuecomment-325610163)). Rather than exposing `runtime` to configuration, I think we need to always set the remote runtime to whatever the local is, so that we upload the code for the right runtime: e6bcbf9ab30299eba7572fdfd388e550a08cf7f1. Sound right?